integration-tests/web/sdk/client/client.go

```diff
@@ -0,0 +1,156 @@
+package client
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"strings"
+
+	"github.com/Khan/genqlient/graphql"
+
+	"github.com/smartcontractkit/chainlink/integration-tests/web/sdk/client/internal/doer"
+	"github.com/smartcontractkit/chainlink/integration-tests/web/sdk/internal/generated"
+)
+
+type Client interface {
```

Comment:
We have similar web routes defined in `core/web` - is there an ability to consolidate/reuse existing implementations?

Reply:
can you link to what implementations you're referring to?